docs/source/documentation.rst

```diff
@@ -131,63 +134,6 @@ AxonDeepSeg can be run via a Graphical User Interface (GUI) instead of the Termi
 
 .. image:: _static/GUI_image.png
 
-To install the GUI, you need to install AxonDeepSeg via Github (see instructions above). If you encounter a problem when installing or using the GUI, please report it on our `issue tracker <https://github.com/neuropoly/axondeepseg/issues>`_.
-FSLeyes is supported on Mac and Linux. Windows users are encouraged to use a virtual machine if they want to use the GUI.
```

Comment:
The FSLeyes plugin doesn't work in my Windows environment. If that is expected, this line should remain in the instructions.
I have the following error while trying to install the plugin.
![Capture](https://user-images.githubusercontent.com/54086142/111663584-56e55f80-87e7-11eb-9249-d227d3b0c89f.PNG)

Reply:
If we wrote that line in the readme, then I'm guessing that's always been the case. Maybe we could open an issue to see if there has been an update such that we could modify the plugin for compatibiltity. @Stoyan-I-A, any insights?

Reply:
FSLeyes isn't supported on Windows. Windows users have two options for running FSLeyes:
1- Use a virtual machine (as I do, which is why I added this line in the documentation on the first place)
2- Use the Linux subsystem (as @mariehbourget does).

Reply:
Done in f36e929